core/src/main/java/com/zextras/carbonio/user_management/controllers/UsersApiController.java

```diff
@@ -40,13 +41,23 @@ public Response getUserInfoByEmail(
 
   public Response getUserInfoById(
     String cookie,
-    UUID uuid,
+    String userId,
     SecurityContext securityContext
   ) {
     Map<String, String> cookies = CookieParser.getCookies(cookie);
 
     return (cookies.containsKey("ZM_AUTH_TOKEN"))
-      ? userService.getInfoById(uuid, cookies.get("ZM_AUTH_TOKEN"))
+      ? userService.getInfoById(userId, cookies.get("ZM_AUTH_TOKEN"))
+      : Response.status(Status.BAD_REQUEST).build();
+  }
+
+  @Override
+  public Response getUsersInfo(String cookie, List<String> userIds, SecurityContext securityContext)
+    throws NotFoundException {
+    Map<String, String> cookies = CookieParser.getCookies(cookie);
+
+    return (cookies.containsKey("ZM_AUTH_TOKEN"))
```

Comment:
```suggestion
    return (cookies.containsKey("ZM_AUTH_TOKEN") && userIds.size() <= MAX_USER_IDS)
```

where `MAX_USER_IDS` is a const declared in this class initialized to `10`